drivers/wifi/wfx200/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_include_directories(${ZEPHYR_BASE}/subsys/net/l2)
+
+if(CONFIG_WIFI_WFX200)
+  zephyr_include_directories(./)
+  zephyr_sources(
+    wfx200.c
+    wfx200_host.c
+    wfx200_events.c
+  )
+endif()
```

Comment:
This should be using a static library instead. See 579ca90e25271a693044667418acef69669f353e.

Reply:
:heavy_check_mark: